connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java

```diff
@@ -372,8 +372,8 @@ public interface ConnectorLogger extends BasicLogger {
      * @param moduleName the module name.
      * @return the message.
      */
-    @Message(id = 41, value = "Failed to load module for driver [%s]")
-    String failedToLoadModuleDriver(String moduleName);
+    @Message(id = 41, value = "Failed to load module for driver [%s]. Location [%s], cause of failure [%s]")
```

Comment:
1) The original message wasn't great even for what it said. 'Failed to load module for driver xxx' implies xxx is the name of the driver. But looking at the code it's the name of the module. So, it should be

"Failed to load driver module [%s]."

2) Why "Location"? The calls to the ModuleLoadException constructors aren't passing in anything that particularly like a location.